content/authors/authors.json

```diff
@@ -0,0 +1,37 @@
+[
+	{
+		"username": "max",
+		"display_name": "Max Zlotskiy",
+		"description": "Max Zlotskiy is a student at Hunter College studying film and computer science. He founded The Icarus in his junior year.",
+		"image": "/images/max.jpg",
```

Comment:
Can you add these images to the PR?

Reply:
I searched my local repo and found that the static assets folder where I keep the pictures wasn't `git add`ed. This tells me that maybe we should be keeping them in the `src/assets` folder, but that has the side effect of bloating the bundle size. Should I get git to track the static folder?

Reply:
I think it's fine to add them in, but maybe we should scale down the images? AFAIK they won't ever be displayed on a large size, so smaller images should create less bloat with no drawback.

Reply:
We can use links to the images instead as was originally planned.

Reply:
@kkhan01 where would we upload the images?

Reply:
Anywhere, my original plan was using github images for some people like `https://github.com/kkhan01.png` or `https://github.com/srafi1.png`. And then I kind of assumed people had images of themselves hosted somewhere, or we can those who really needed to be added.